microservice/src/test/java/com/skytouch/MicroserviceApplicationTests.java

```diff
@@ -0,0 +1,61 @@
+package com.skytouch;
+
+
+import com.skytouch.common.model.Product;
+import com.skytouch.model.ProductEntity;
+import com.skytouch.repository.ProductRepository;
+import com.skytouch.service.ProductService;
+import org.dozer.DozerBeanMapper;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import java.util.List;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class MicroserviceApplicationTests {
+
+    @Autowired
+    private ProductService productService;
+    @Autowired
+    private ProductRepository productRepository;
+
+    @Autowired
+    private DozerBeanMapper mapper;
+
+    private Product product;
+    private ProductEntity productEntity;
+
+    @Before
+    public void setupData() {
+        product = new Product("insertTest", "Unit Test", 1, 1);
+        productEntity = mapper.map(product, ProductEntity.class);
+        mapper = new DozerBeanMapper();
+
+    }
+
+    @Test
+    public void getAllProducts() {
+        List<ProductEntity> products = productRepository.getProducts();
```

Comment:
Are you only testing the quantity of elements?. Shouldn't you also test the elements are the same?

Reply:
Sorry, my mistake. The test was modify to evaluate that all values are equals in the method: testGetAllProductList in the class ProductServiceTests.